### PR TITLE
doc: getting started: add Arch Linux package for Zephyr SDK

### PR DESCRIPTION
**Pull request by alexeicolin**
Package was tested by building the Hello World sample for the QEMU Cortex-M3 machine and running it.

(Ideally, Zephyr SDK would be packaged for other distros as well, and some of the details in this guide, such as dependency installation would no longer need to be manually performed by the user and could be dropped from the guide for conciseness and fewer content to maintain.)

### CODE REVIEW

File: doc/getting_started/installation_linux.rst [hunk 1]
```diff
@@ -10,6 +10,7 @@ applications on the following Linux distributions:
 
 * Ubuntu 16.04 LTS 64-bit
 * Fedora 25 64-bit
+* Arch Linux (install [zephyr-sdk](https://aur.archlinux.org/packages/zephyr-sdk) package from AUR)
```
Comment (dbkinder) [inline doc/getting_started/installation_linux.rst]: That's wiki syntax for links.  In reST it should be:

    * Arch Linux (install `zephyr-sdk <https://aur.archlinux.org/packages/zephyr-sdk>`_ package from AUR)

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Review by dbkinder — APPROVED** [review]
That works. Thanks.

**Comment by dbkinder** [comment]
... but the commit message still needs a body (can't just be the subject line)

**Comment by alexeicolin** [comment]
Added explanatory commit message.

**Review by nashif — APPROVED** [review]

**Pull request merged**